src/enums.js

```diff
@@ -98,6 +107,27 @@ const populateErrorTypes = (errorCodes, errorTypes) => {
   return newErrorCodes
 }
 
+/**
+ * Returns an object representing a Mojaloop API spec error code combined with overrides
+ *
+ * @param errorCodes {object} - Mojaloop API spec error code enums
+ * @param override {object} - central-services-error-handling defined override enum
+ * @returns {object} - Object representing the Mojaloop API spec error enums with associated types
+ */
+const populateOverrides = (errorCodes, override) => {
```

Comment:
I am not sure we should be overriding existing ML-API codes. We should only be allowed to add custom codes.

Reply:
I agree. See my reply on previous comment.

Reply:
This has been resolved as discussed earlier today. Please, review latest changes.